rc/servers.kak

```diff
@@ -244,6 +244,14 @@ hook -group lsp-filetype-javascript global BufSetOption filetype=(?:javascript|t
     #     codeAction.disableRuleComment = { enable = true, location = "separateLine" }
     #     codeAction.showDocumentation = { enable = false }
     # }
+    # set-option buffer lsp_servers %{
+    #     [tailwindcss-language-server]
+    #     root_globs = ["tailwind.config.*"]
+    #     command = "tailwindcss-language-server"
+    #     args = ["--stdio"]
+    #     [tailwindcss-language-server.settings.tailwindcss-language-server]
+    #     emmetCompletions = true
```

Comment:
do these settings work?
Our README has an example that uses `[tailwindcss-language-server.settings.tailwind]`
Probably only one of the two is correct; we send only whichever section the server asks for
(unless `settings_section` is given).

Reply:
Sorry I should not have added that, I was testing it in my personal config and must have copied it over.[1] I've gone ahead and removed some of those unnecessary parts.
<details><summary>[1]</summary>
<p>
Curiously though I was unable to get that setting to work with either <code>settings.tailwind</code> or <code>settings.tailwindCSS</code> <a href="https://github.com/tailwindlabs/tailwindcss-intellisense#extension-settings">as shown here<a/>
</p>
</details>

Reply:
`settings.tailwindCSS` should work, judging by the logs

```
LSP: DEBG From server tailwindcss-language-server: {"jsonrpc":"2.0","id":5,"method":"workspace/configuration","params":{"items":[{"section":"tailwindCSS","scopeUri":"file:///home/johannes/git/kakoune-lsp/t/test.js"}]}}
LSP: DEBG To server tailwindcss-language-server: {"jsonrpc":"2.0","result":[{"emmetCompletions":true}],"id":5}
```

but I don't know how to test it, LMK if you know an example